packages/client/test/relayClient.test.ts

```diff
@@ -52,7 +52,6 @@ beforeEach(async () => {
   crypto.getRandomValues(keyCard);
   relayClient = new RelayClient({
     relayEndpoint,
-    useTLS: false,
```

Comment:
don't need to specify this, this is determined from relay endpoint in the client